src/core/server/services/comments/pipeline/phases/tagStaff.spec.ts

```diff
@@ -0,0 +1,221 @@
+import {
+  GQLFEATURE_FLAG,
+  GQLTAG,
+  GQLUSER_ROLE,
+} from "coral-server/graph/schema/__generated__/types";
+import { Story } from "coral-server/models/story";
+import { Tenant } from "coral-server/models/tenant";
+import { User } from "coral-server/models/user";
+import {
+  createSiteFixture,
+  createStoryFixture,
+  createTenantFixture,
+  createUserFixture,
+} from "coral-server/test/fixtures";
+import { IntermediateModerationPhaseContext } from "..";
+import { tagStaff } from "./tagStaff";
+
+interface Case {
+  author: User;
+  story: Story;
+  tenant: Tenant;
+}
+
+/**
+ * testCase returns how a comment would
+ * be tagged for a given author, story, and tenant
+ */
+const testCase = (input: Case): GQLTAG | void => {
+  const result = tagStaff(
+    (input as unknown) as IntermediateModerationPhaseContext
+  ) as any;
+
+  if (result?.tags?.length > 1) {
+    throw new Error("tagStaff unexpectadly applied more than one tag");
+  }
+
+  return result?.tags?.[0];
+};
+
+const tenant = createTenantFixture();
+const tenantID = tenant.id;
+
+// TODO: this will need to be updated once https://github.com/coralproject/talk/pull/3769 is merged
```

Comment:
This will be a very fast fix, but the way this feature is indicated will need to be updated once @kabeaty 's PR is in.

Reply:
I've approved, but put a `don't merge` tag on this cause it sounds like you want this other PR to get merged first.